src/app/modules/group/components/managed-group-list/managed-group-list.component.ts

```diff
@@ -16,29 +14,23 @@ export class ManagedGroupListComponent implements OnDestroy, OnInit {
 
   data: Group[] = [];
 
-  private dataFetching = new Subject<{}>();
+  private subscription?: Subscription;
 
   constructor(private managedGroupService: ManagedGroupsService) {
-    this.dataFetching.pipe(
-      switchMap(_ => merge(
-        of(fetchingState()),
-        this.managedGroupService.getManagedGroups().pipe(map(readyState)),
-      )),
-    ).subscribe(
-      state => {
-        this.state = state.tag;
-        if (isReady(state)) this.data = state.data;
-      },
-      _err => this.state = 'error',
-    );
   }
 
   ngOnInit(): void {
-    this.dataFetching.next();
+    this.subscription = this.managedGroupService.getManagedGroups().pipe().subscribe(
```

Comment:
There is no subscription to keep on an http service, it cannot last forever.

Reply:
Probably you mixed with placed where there is a subscription to be able to cancel it when a refresh occurs.

Reply:
Fixed.